.github/actions/linux-release-builder/Dockerfile

```diff
@@ -2,9 +2,8 @@ FROM phusion/holy-build-box-64:latest
 
 # Install a copy of git and the Boost headers
 # curl -L follows redirects and -k ignores SSL certificate warnings
-RUN yum install -y rh-git29 devtoolset-8 \
-    && curl -Lk -o boost.tar.gz https://sourceforge.net/projects/boost/files/boost/1.70.0/boost_1_70_0.tar.gz/download \
-    && tar --strip-components=1 -xf boost.tar.gz boost_1_70_0/boost \
+RUN curl -Lk -o boost.tar.gz https://sourceforge.net/projects/boost/files/boost/1.72.0/boost_1_72_0.tar.gz/download \
```

Comment:
do we not need the yum install lines any more?

Reply:
The new base image has devtoolset-8 installed and a copy of git.